test/envtest/sgxdeviceplugin_controller_test.go

```diff
@@ -20,20 +20,25 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	apps "k8s.io/api/apps/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 
 	devicepluginv1 "github.com/intel/intel-device-plugins-for-kubernetes/pkg/apis/deviceplugin/v1"
+	"github.com/intel/intel-device-plugins-for-kubernetes/pkg/controllers"
 )
 
 var _ = Describe("SgxDevicePlugin Controller", func() {
 
 	const timeout = time.Second * 30
 	const interval = time.Second * 1
 
-	AfterEach(func() {
-		time.Sleep(time.Second * 2)
+	BeforeEach(func() {
+		up("sgx", false)
```

Comment:
Not sure we want to test all controllers isolated from each other in integration tests. The expected setup is that users run the operator with multiple controllers enabled.